### PR TITLE
Feat: fullscreen cards

### PR DESCRIPTION
**Pull request by micwallace**
Hey @futantan, this PR adds support for fullscreen tokenscript cards as well as slightly increasing the width of a normal card to be in line with the width used by TS viewer.

### CODE REVIEW

File: apps/extension/components/iframe-popup.tsx [hunk 1]
```diff
@@ -2,12 +2,13 @@ import { VisuallyHidden } from "@radix-ui/react-visually-hidden"
 import "@repo/tlinks/index.css"
 import { CircleX, SquareArrowUpRight } from "lucide-react"
 import { forwardRef, useImperativeHandle, useState } from "react"
+import {TokenscriptCardMetadata} from "@repo/tlinks/src/utils/fetch-ts-data.ts";
```
Comment (futantan) [inline apps/extension/components/iframe-popup.tsx:5]: The path of the import is not correct.
you can treat `@repo/tlinks` as a npm package, the `src` folder will not be included

**Comment by futantan** [comment]
hi @micwallace can you share a tokenscript viewer link to let me test the `fullScreen` feature?

**Comment by micwallace** [comment]
@futantan you can use this one: https://x.com/micwallace11/status/1857265021111431461

File: apps/extension/entrypoints/twitter-observer.content/twitter-observer.tsx [hunk 2]
```diff
@@ -3,9 +3,12 @@ import { setupTwitterObserver } from "@repo/tlinks/ext/twitter"
 import "@repo/tlinks/index.css"
 import { useEffect, useState } from "react"
 import "~/assets/style.css"
+import {TokenscriptCardMetadata} from "@repo/tlinks/dist/utils/index";
```
Comment (futantan) [inline apps/extension/entrypoints/twitter-observer.content/twitter-observer.tsx:6]: please update the import path here, can treat `@repo/tlinks` as a normal npm package

**Review by futantan — COMMENTED** [review]
the path of `@repo/tlinks` need to be updated, others looks good to me

**Comment by futantan** [comment]
please help to fix the build error above 👆🏻

**Comment by futantan** [comment]
> @futantan you can use this one: https://x.com/micwallace11/status/1857265021111431461

it looks awesome! 
One feedback is the food in the game is a bit too small and hard to spot.

File: apps/extension/entrypoints/twitter-observer.content/twitter-observer.tsx (hunk 2 above)
Comment (micwallace) [reply apps/extension/entrypoints/twitter-observer.content/twitter-observer.tsx]: @futantan should be okay now I think?

**Comment by micwallace** [comment]
@futantan The vercel build error? It says "This Deployment has been disabled. "

**Comment by micwallace** [comment]
Thanks for the feedback, I agree it's too small for small screens especially.

**Comment by futantan** [comment]
> @futantan The vercel build error? It says "This Deployment has been disabled. "

hi @micwallace you can find the error via `npm run build` in the root path of this project

**Comment by micwallace** [comment]
@futantan All fixed

**Pull request merged**